Translations/cs.lproj/Credits.strings

```diff
@@ -1,15 +1,15 @@
 "_PROJECTTITLE_" = "Projekt Keka";
 "_PROGRAMMER_" = "Programátor, designér &amp; maintainer";
-"_ARTWORK_" = "Grafický designér";
+"_ARTWORK_" = "Designéři";
 "_TRANSLATIONS_" = "Překladatelé";
 "_TRANSLATIONSOLD_" = "Překladatelé (verze 0.1-0.1.4.3)";
 "_TESTERS_" = "Občasní testeři";
 "_DONATIONS_" = "Příspěvky (většina seřazená podle data)";
 "_OPENSOURCE_" = "Keka využívá otevřený kód";
 "_FREEWARE_" = "Keka využívá freeware binární soubory";
 "_ONEMORETHING_" = "A ještě něco";
-"_THANKS_" = "Děkujeme, že používáte Keka!";
-"_SOURCE_" = "Kvůli licenčním problémům nebude zdrojový kód Keka 1.0 zveřejněn. Jestli budete potřebovat jakoukoliv pomoc ohledně licence, obraťte se na info@keka.io nebo na stránku Projekt na oficální stránce aplikace Keka. Uvítáme jakoukoliv pomoc.";
+"_THANKS_" = "Děkujeme, že používáte Keku!";
```

Comment:
I'm not sure changing Keka, being it the product name, is a good idea. Is this a common practice in Apple's macOS bundled apps?

Reply:
I looked into the Czech translation memory for macOS and they use declension like this when the resulting word isn't too different from the original. I think that in this case, it's obvious what this changed word is referring to. This is why I didn't change it in occurrences where the final word would have more than one letter different (e.g. "Kece").

I think that in this case, the changed word helps the text flow better, but it's no big deal, if you think it shouldn't be changed, it doesn't have to be 👍

Reply:
So forgive my lack of knowledge but you will use those declensions in own names?

Like: `Děkujeme, že používáte Davidu!`

Reply:
It's okay, Czech is extremely complex 😦 Yes, every single noun, including names, has seven possible declensions. In the case of David, this sentence would be Děkujeme, že používáte Davida! (because Keka is feminine and David is masculine).

I accidentally somehow managed to start a review, I'm sorry for that, Github is still very confusing to me 😅